.github/workflows/unit-integration-test.yml

```diff
@@ -0,0 +1,109 @@
+        env:
+          AWS_REGION: ${{ vars.AWS_REGION }}
+          ECR_REPO_ENV_AWS_ACCOUNT_NUMBER: ${{ vars.SBX_ACCOUNT_NUMBER }}
+        with:
+          params: |
+            ARTIFACTORY_URL=/artifactory/url
+            ARTIFACTORY_USER=/artifactory/user
+            ARTIFACTORY_PASSWORD=/artifactory/password
+            CC_TEST_REPORTER_ID=cc-test-reporter-id
+
+      - name: Assume role in target account
+        uses: aws-actions/configure-aws-credentials@v3
+        with:
+          aws-region: us-east-1
+          role-to-assume: arn:aws:iam::${{ vars.DEV_ACCOUNT_NUMBER }}:role/delegatedadmin/developer/github-actions-runner-role
+
+      - name: Set env vars from AWS params
+        uses: cmsgov/ab2d-bcda-dpc-platform/actions/aws-params-env-action@main
+        env:
+          AWS_REGION: ${{ vars.AWS_REGION }}
+        with:
+          params: |
+            OKTA_CLIENT_ID=/okta/client-id
+            OKTA_CLIENT_PASSWORD=/okta/client-secret
+            SECONDARY_USER_OKTA_CLIENT_ID=/secondary-okta/client-id
+            SECONDARY_USER_OKTA_CLIENT_PASSWORD=/secondary-okta/client-secret
+
+      - name: Remove Maven Directory
+        run: |
+          if [ -d "/opt/maven/apache-maven-3.6.3" ]; then
+            sudo rm -rf /opt/maven/apache-maven-3.6.3
+            echo "Maven directory removed."
+          else
+            echo "Maven directory does not exist."
+          fi
+
+      - name: Install Maven 3.6.3
+        run: |
+          wget https://archive.apache.org/dist/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz
+          tar xzvf apache-maven-3.6.3-bin.tar.gz
+          sudo mv apache-maven-3.6.3 /opt/maven
+          echo "PATH=$PATH:/opt/maven/bin" >> $GITHUB_ENV
+
+      - name: Create ab2d workspace directory and copy in keystore
+        run: sudo mkdir -p "$WORKSPACE/opt/ab2d"
+
+      - name: Download Code Coverage
+        run: |
+          mkdir -p codeclimate
+          if [ ! -f ./codeclimate/cc-test-reporter ]; then
+            curl -L --no-progress-meter https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 \
+            > ./codeclimate/cc-test-reporter && chmod +x ./codeclimate/cc-test-reporter
+          fi
+
+      - name: Set directory permissions
+        run: |
+          sudo chmod -R 777 /opt/actions-runner/_work/ab2d/ab2d/common/
+          # Add other commands if necessary
+
+      - name: Clean maven
+        run: |
+          mvn --version
+          echo $WORKSPACE
+          mvn -U clean
+
+      - name: Run unit and integration tests
+        run: |
+          mvn -s settings.xml -X -Dusername=${ARTIFACTORY_USER} -Dpassword=${ARTIFACTORY_PASSWORD} -Drepository_url=${ARTIFACTORY_URL} test -pl common,job,coverage,api,worker
+        working-directory: ${{ github.workspace }}
+
+      - name: Cleanup -
+        run: |
+          sudo rm -rf /opt/actions-runner/_work/ab2d/ab2d/opt/ab2d
```

Comment:
Use `${{ github.workspace }}/opt/ab2d` instead of hardcoding the full path.

Also, this step should include an `if: ${{ !cancelled() }}` to run on both success and failure. See https://docs.github.com/en/actions/learn-github-actions/expressions#always